### PR TITLE
AO-12: Update documentation to point to Add Ons instead of Modulus

### PR DESCRIPTION
**Pull request by f4ww4z**
https://issues.openmrs.org/browse/AO-12

I fixed https://modules.openmrs.org links which should be updated to the new https://addons.openmrs.org/ .

### CODE REVIEW

File: en/Case_study/yourFirstModule.md [hunk 1]
```diff
@@ -451,6 +451,6 @@ Once your module is released, you may think that your work is over. However, the
 
 When done with developing and testing your module, you can release it for developers by deploying to the Maven repository using instructions at: http://om.rs/newdevtagging
 
-For end users, you can upload it to the module repository http://modules.openmrs.org which is available to everyone. Read more about our rules and regulations for it here: http://om.rs/newdevmodrepo 
+For end users, you can upload it to the [Add-ons index](https://addons.openmrs.org) which is available to everyone. Read more about our rules and regulations for it here: http://om.rs/newdevmodrepo
```
Comment (dkayiwa) [inline en/Case_study/yourFirstModule.md]: Did you intentionally drop the words "module repository"?

File: en/Case_study/yourFirstModule.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ To put this book in perspective, we'll walk through a fictional scenario that re
 
 Let's suppose there is an established health clinic called **Amani Clinic** in East Africa with a few staff members who are experienced in ICT and medical informatics. They are very talented and very busy. You find and read a ticket they created when they realized functionality for adding, editing, saving, and listing of departments would be helpful to their implementation. For the purposes of this task, we will define a department as a hospital ward designed to perform a specific purpose.
 
-Someone has left a comment ***"This would be a good module project!"* **on the ticket. You would love a new project to work on and happen to have an interest in the larger issue of "departments" as they relate to health informatics, so you are interested in taking up this task.
+Someone has left a comment **"This would be a good module project!"** on the ticket. You would love a new project to work on and happen to have an interest in the larger issue of "departments" as they relate to health informatics, so you are interested in taking up this task.
```
Comment (dkayiwa) [inline en/Case_study/yourFirstModule.md]: What is the change here?

File: en/Case_study/yourFirstModule.md (hunk 1 above)
Comment (f4ww4z) [reply en/Case_study/yourFirstModule.md]: Yes I did @dkayiwa . Now, I think it's wrong as you cant actually upload to add-ons index. Should I change this?

File: en/Case_study/yourFirstModule.md (hunk 2 above)
Comment (f4ww4z) [reply en/Case_study/yourFirstModule.md]: I'll revert this I think.

Comment (dkayiwa) [reply en/Case_study/yourFirstModule.md]: So did you revert?

Comment (f4ww4z) [reply en/Case_study/yourFirstModule.md]: I have reverted.

**Comment by f4ww4z** [comment]
@dkayiwa , I added (and corrected) some more info regarding the Add-ons index. Please take a look :)

**Comment by dkayiwa** [comment]
@rkorytkowski what do you think of this? 😊

**Pull request merged**